docs/book/src/cronjob-tutorial/testdata/project/controllers/suite_test.go

```diff
@@ -132,10 +139,6 @@ var _ = BeforeSuite(func() {
 		err = k8sManager.Start(ctrl.SetupSignalHandler())
 		Expect(err).ToNot(HaveOccurred())
 	}()
-
-	k8sClient = k8sManager.GetClient()
-	Expect(k8sClient).ToNot(BeNil())
```

Comment:
imo; we should do it as we do in the scaffolds (k8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})). See: https://github.com/kubernetes-sigs/kubebuilder/blob/master/testdata/project-v3/controllers/suite_test.go

Could you please update it for it be compatible with how we scaffold it?

Reply:
Ok, so should I delete the whole manager part (from line 105 to line 141) and add my changes in the comment on line 99?

Reply:
@camilamacedo86 it is exactly done as described: `8sClient, err = client.New(cfg, client.Options{Scheme: scheme.Scheme})` in line 101 of this same file. It is just hidden by GitHub.